modules/signals/vite.config.mts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="vitest" />
+
+import angular from '@analogjs/vite-plugin-angular';
+
+import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin';
+
+import { defineConfig } from 'vite';
+
+// https://vitejs.dev/config/
+export default defineConfig(({ mode }) => {
+  return {
+    root: __dirname,
+    plugins: [
+      angular(),
+      nxViteTsPaths(),
+    ],
+    test: {
+      globals: true,
+      pool: 'forks',
+      environment: 'jsdom',
+      setupFiles: ['test-setup.ts'],
+      include: ['**/*.spec.ts'],
+      exclude: ['migrations/**', 'schematics/**'],
```

Comment:
Is there a possibility of executing the migrations/schematics tests with Vitest as well and getting rid of `signals/jest.config`?

Reply:
Yea, I pushed a commit to build the signals package and use the build output for schematics/migrations tests and deleted the `signals/jest.config.ts`. Now everything is running on Vitest.